### ISSUE TITLE
val and test set in FC100

### ISSUE THREAD

**Issue by FrigEgg** [kjunelee/MetaOptNet#15]
Thanks for your code! I think it is really great work. But I have a problem about FC100. During  the training, the training loss is not problem， gradually decrease as the number of epoch increases. However, accuracy on val and test set remains constant around 50%. 
Here is the version of my package
Python 3.6.3
torch                   1.0.1.post2 
torchnet                0.0.4 
qpth                    0.0.13

**Comment by kjunelee** [comment]
Hello. Thanks for your interest in our work.

I think your result seems okay as the performance of our system for FC100 is around 40 % for 1-shot and 55 % for 5-shot as shown in Table 2 of our paper. As a note, you will observe a jump in performance at epoch 21 where we reduce the learning rate to 1/10.

**Comment by FrigEgg** [comment]
Thank you for your reply! I see what your mean. I will try something else.。
